models/zoo.yaml

```diff
@@ -34,3 +34,5 @@ whisper_large:
   url: oss://speechio-leaderboard/models/whisper_large/
 whisper_large_v2:
   url: oss://speechio-leaderboard/models/whisper_large_v2/
+ximalaya_api_zh:
```

Comment:
remove this entry, ur submission is light-weight client, so there is no need to store them in OSS, we keep them in this github repo.